cmd/crossplane/main.go

```diff
@@ -37,48 +36,57 @@ import (
 	"github.com/crossplane/crossplane/cmd/crossplane/rbac"
 )
 
+type debugFlag bool
+
+var cli struct {
+	Debug debugFlag `short:"d" help:"Print verbose logging statements."`
+
+	Core core.Command `cmd:"" help:"Start core Crossplane controllers." default:"1"`
+	Rbac rbac.Command `cmd:"" help:"Start Crossplane RBAC Manager controllers."`
+}
+
+func (d debugFlag) BeforeApply(ctx *kong.Context) error { // nolint:unparam
```

Comment:
Could you include a comment explaining why we need to disable the `unparam` linter here please.